main.js

```diff
@@ -23,6 +23,18 @@ $(document).ready(function () {
     }, 0);
   }
 
+  $('select').on('change', function () {
+    if ($(this).val() == 'epilepsy') {
+      $('body').removeClass().addClass('theme-epilepsy');
```

Comment:
You are removing all the body classes here, you should take into account that there can be other classes, possibly added in the future. You should remove only those with `theme-` prefix